pkg/imported/components/CruImported.vue

```diff
@@ -243,6 +281,37 @@ export default defineComponent({
         delete this.normanCluster.localClusterAuthEndpoint.fqdn;
       }
     },
+    async done() {
+      if (!this.isEdit) {
+        await this.normanCluster.waitForProvisioning();
```

Comment:
When I create a new imported cluster there's a couple-second delay between when the save button turns green and when I am redirected to the cluster registration view. It's not a big delay but it is noticeably different from other cluster provisioning. I think if you move this await to `actuallySave` the save button behaves more in line with other cluster provisioning: there's a spinning wheel until the page is ready to redirect


```
    async actuallySave() {
      if (this.isEdit) {
        return await this.normanCluster.save();
      } else {
        await this.normanCluster.save();

        return await this.normanCluster.waitForProvisioning();
      }
    }
```